x/evm/keeper/state_transition.go

```diff
@@ -110,8 +113,8 @@ func (k Keeper) VMConfig(ctx sdk.Context, msg core.Message, params types.Params,
 		Debug:       debug,
 		Tracer:      tracer,
 		NoRecursion: false, // TODO: consider disabling recursion though params
-		NoBaseFee:   fmParams.NoBaseFee,
-		ExtraEips:   params.EIPs(),
+		NoBaseFee:   noBaseFee,
```

Comment:
Actually, I searched in go-ethereum's code, the only place this field is used is here: https://github.com/ethereum/go-ethereum/blob/master/core/state_transition.go#L238, which is not used by us.
So maybe we can hardcode a `true` here to save a query? @fedekunze